src/main/java/it/pagopa/transactions/commands/handlers/TransactionActivateHandler.java

```diff
@@ -230,6 +222,34 @@ public Mono<Tuple2<Mono<TransactionActivatedEvent>, String>> handle(
         );
     }
 
+    private void spanCustomTransaction(PaymentRequestInfo paymentRequestInfo) {
+        String transactionActivationDateString = paymentRequestInfo.activationDate();
+        Duration paymentTokenValidityTimeLeft = Duration.ofSeconds(paymentTokenTimeout);
```

Comment:
Isn't this the the payment token validity time? (i.e. should be the same as the first activation, not less)

Reply:
The paymentTokenTimeout, valued as class attribute, contains the `payment.token.validity` configured value

Reply:
Ok, I would remove `left` in the name then, what do you think?